appveyor.yml

```diff
@@ -47,7 +49,14 @@ build_script:
 # Build and test onnx.
 - cmd: cd c:\projects\onnx
 - cmd: set ONNX_BUILD_TESTS=1
-- cmd: set CMAKE_ARGS="-DONNX_USE_PROTOBUF_SHARED_LIBS=ON"
+#The latest libprotobuf Windows packages in anaconda was built with          
+#        -Dprotobuf_BUILD_SHARED_LIBS=ON 
+#        -Dprotobuf_MSVC_STATIC_RUNTIME=OFF
+#We must align with them. We must set USE_MSVC_STATIC_RUNTIME and 
+#ONNX_USE_PROTOBUF_SHARED_LIBS based on their config
+- cmd: set USE_MSVC_STATIC_RUNTIME=0
+# Don't put quotation marks in the following line
```

Comment:
Is this comment needed?

Reply:
Take a look of the old line. It's wrong. It shouldn't have the mark.

Reply:
With the  quotation mark, it won't work as expected.